app/portainer/models/hostmanagement/fdo.ts

```diff
@@ -0,0 +1,12 @@
+export interface FDOConfiguration {
```

Comment:
featuer folder layout - models, services, controllers should be grouped together.